rosidl_gen/generator_primitive.js

```diff
@@ -25,6 +26,16 @@ const rosidl_generator_nodejs__String = StructType({
   capacity: ref.types.size_t,
 });
 
+const libstd_msgs__rosidl_generator_c = ffi.Library('libstd_msgs__rosidl_generator_c', {
+  'rosidl_generator_c__String__assign': [ref.types.bool, [ref.refType(rosidl_generator_nodejs__String), ref.types.CString]],
+});
+
+function ffi_rosidl_generator_nodejs__String__assign(buf, text) {
+  let strPtr = new Buffer(text + '\0');
+  strPtr.type = ref.types.CString;
+  libstd_msgs__rosidl_generator_c.rosidl_generator_c__String__assign(buf, strPtr);
```

Comment:
In the final version, we should do our own function bindings instead of FFI, which is faster, and recyclable (we can have a chance to free memory in C++ destructor)